index.js

```diff
@@ -0,0 +1,158 @@
+const Alexa = require('ask-sdk-core')
+const axios = require('axios')
+
+const LaunchRequestHandler = {
+  canHandle(handlerInput) {
+    return (
+      Alexa.getRequestType(handlerInput.requestEnvelope) === 'LaunchRequest'
+    )
+  },
+  handle(handlerInput) {
+    const speakOutput = 'The truth about Cats and Dogs'
+    return handlerInput.responseBuilder
+      .speak(speakOutput)
+      .reprompt(speakOutput)
+      .getResponse()
+  }
+}
+async function getFacts(animal) {
+  if (animal === 'dogs') {
+    animal = 'dog'
+  }
+  if (animal === 'cats') {
+    animal = 'cat'
+  }
```

Comment:
Fun little thing you can do here is use something called a Regex https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions

```JavaScript
const re = /s$/
animal = animal.replace(re, '')
```